src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -394,4 +427,55 @@ public V setValue(final V value) {
     return result;
   }
 
+  /**
+   * A Serializer/Deserializer/Serde implementation for use when you know the data is always null
+   * @param <T> The type of the stream (you can parameterize this with any type,
+   *           since we throw an exception if you attempt to use it with non-null data)
+   */
+  static class NothingSerde<T> implements Serializer<T>, Deserializer<T>, Serde<T> {
+
+    private final boolean strict;
+
+    NothingSerde(final boolean strict) {
```

Comment:
This lets you prove your assumption that it'll never be used with real data, or just drop any non-null data, depending on your test case.

Reply:
Shouldn't this be the only way to use it? If it's not `strict`, it seem using this Serde would not make sense? Can you give an example when it make sense to use it non strict?

Reply:
Good feedback. I did have one use case, but I've refactored it out. So, now this parameter is just tech debt. I'll remove it.